lldb/test/API/source-manager/TestSourceManager.py

```diff
@@ -323,13 +323,13 @@ def test_artificial_source_location(self):
         )
 
         self.expect(
-            "run",
-            RUN_SUCCEEDED,
+            "thread info", substrs=[f"{src_file}:0", "stop reason = breakpoint"]
+        )
+        self.expect(
+            "frame select 0",
             substrs=[
-                "stop reason = breakpoint",
-                "%s:%d" % (src_file, 0),
-                "Note: this address is compiler-generated code in " "function",
-                "that has no source code associated " "with it.",
+                "Note: this address is compiler-generated code in function",
+                "that has no source code associated with it.",
```

Comment:
Can't this be a single `process status` command instead of both `thread info` & `frame select 0` ?

Reply:
Just tried it out, it totally can be. Thanks for the tip!